make/modules/java.desktop/lib/ClientLibraries.gmk

```diff
@@ -412,14 +293,6 @@ else
      HARFBUZZ_CFLAGS += -DHAVE_INTEL_ATOMIC_PRIMITIVES -DHB_NO_VISIBILITY
    endif
 
-   # Early re-canonizing has to be disabled to workaround an internal XlC compiler error
-   # when building libharfbuzz
-   ifeq ($(call isTargetOs, aix), true)
-    ifneq ($(TOOLCHAIN_TYPE), clang)
-     HARFBUZZ_CFLAGS += -qdebug=necan
-    endif
-   endif
```

Comment:
This needs to remain (in `ClientLibraries.gmk`) for AIX builds with xlc.

Reply:
Restored